packages/taikoon-ui/src/lib/connect/web3modal.ts

```diff
@@ -29,13 +30,13 @@ export const web3modal = createWeb3Modal({
   featuredWalletIds: [],
   allowUnsupportedChain: true,
   excludeWalletIds: ['c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96'],
-  chainImages,
+  chainImages: [],
```

Comment:
we should configure chain images

Reply:
Disabled for SSR configs, bringing those back!